xen/arch/arm/irq.c

```diff
@@ -32,19 +34,43 @@ const unsigned int nr_irqs = NR_IRQS;
 static unsigned int local_irqs_type[NR_LOCAL_IRQS];
 static DEFINE_SPINLOCK(local_irqs_type_lock);
 
+#define GSX_MAX_OS_CNT                                     8
```

Comment:
this is the second time we define this constant...

Reply:
Yes, you are right (xen + libxl). For keeping it in a single place, we need to define it somewhere in a public header. Shall I do that?